### ISSUE TITLE
Value not found in DB for page-break-inside

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/sprints#2880]
On https://developer.mozilla.org/en-US/docs/Web/CSS/page-break-inside there's likely an error message:

```
Media     as each of the properties of the shorthand:
          * visual: Value not found in DB!
          * paged: Value not found in DB!
```

I've noticed the warning at the top, that this property was replaced by [break-inside](https://developer.mozilla.org/en-US/docs/Web/CSS/break-inside) but wanted to highlight, that maybe some house cleaning is needed here :-)

**Comment by chrisdavidmills** [comment]
I think this problem is occuring for any properties that have an array of multiple values for `media`, rather than just a single value. Here's the JSON data for `page-break-inside`: https://github.com/mdn/data/blob/master/css/properties.json#L7195

Array values for `media` validate in this repo, but the macro that puts that data on the page seems to make a mess of it when `media` is an array.

We've just not gotten round to fixing it. @wbamberg do you think this would be a quick fix, or given that we are trying to remove all macros from MDN, should we just not worry about this?

**Comment by wbamberg** [comment]
> do you think this would be a quick fix

The `{{CSSInfo}}` macro is complex and has no tests. No changes to it are both quick and safe :).

If we did want to tackle it, the correct thing would be to remove `media` entirely across the board, as per the discussion here: https://discourse.mozilla.org/t/updates-to-the-css-info-boxes/30855.

As far as I understand the CSSInfo macro, just removing `media` from the data would be sufficient to stop it showing up in the tables.

> given that we are trying to remove all macros from MDN, should we just not worry about this?

Some time in the future we're going to decide what to do about CSS info boxes. Really, for the sake of the future, I'd like us to:

* decide what we actually want in CSS info boxes, with a view to making it much simpler and meeting actual user needs (for example, the Discourse thread linked above fingers `media` and `Canonical order`, but the [recent conversation about `percentage`](https://github.com/mdn/data/pull/409) makes me question whether that adds any value over the free text in the page describing how percentages are applied).
* reimplement the CSSInfo macro from the ground up to support only what we need.
* remove data from mdn/data that isn't meeting that need

That would put us in a better place to migrate the CSS content.

**Comment by wbamberg** [comment]
> The {{CSSInfo}} macro is complex and has no tests. No changes to it are both quick and safe :).

I might be wrong about this. https://github.com/mdn/kumascript/pull/1356 is definitely quick and possibly also safe :).

**Comment by Ryuno-Ki** [comment]
Happy to hear that :-)

**Comment by wbamberg** [comment]
I've merged https://github.com/mdn/kumascript/pull/1356, Ryan made a new deployment today and I've refreshed all 1007-odd CSS pages, and "Media" and "Canonical order" are both gone from these boxes.

Since there are no longer errors in the page for [`page-break-inside`](https://developer.mozilla.org/en-US/docs/Web/CSS/page-break-inside), I'm closing this.

I'd still like to remove the underlying data from mdn/data, but that's another story.

Thanks for filing, @Ryuno-Ki , and finally giving me the motivation to get this fixed :).
